### PR TITLE
testing ci change

### PR DESCRIPTION
**Pull request by stephenwaite**
think Brady meant can see the errors with these changes and a PR, #2518

### CODE REVIEW

**Comment by stephenwaite** [comment]
so the problem is at https://github.com/openemr/openemr/blob/f42ff759247466237274bf6fcfc42ad9c7a66abc/ci/ci.sh#L20-L23

`$ bash ci/ci.sh --dir ./
PHP Warning:  PHP Startup: Unable to load dynamic library 'ldap.so' (tried: /home/travis/.phpenv/versions/7.3.2/lib/php/extensions/no-debug-zts-20180731/ldap.so (/home/travis/.phpenv/versions/7.3.2/lib/php/extensions/no-debug-zts-20180731/ldap.so: cannot open shared object file: No such file or directory), /home/travis/.phpenv/versions/7.3.2/lib/php/extensions/no-debug-zts-20180731/ldap.so.so (/home/travis/.phpenv/versions/7.3.2/lib/php/extensions/no-debug-zts-20180731/ldap.so.so: cannot open shared object file: No such file or directory)) in Unknown on line 0`

**Comment by stephenwaite** [comment]
that fixed the ldap error but something must have changed with squizlabs?

**Comment by stephenwaite** [comment]
squizlabs 3.1* and 3.2.* failing
`Fatal error: Uncaught PHP_CodeSniffer\Exceptions\RuntimeException: "continue" targeting switch is equivalent to "break"`

**Comment by stephenwaite** [comment]
for php 7.3 squizlabs needs to be > 3.3.0, https://github.com/squizlabs/PHP_CodeSniffer/issues/2319

**Comment by stephenwaite** [comment]
looking at the first of hundreds of travis errors :), looks like the ignoreIndentationTokens could be updated to include T_CLOSE_TAG

**Comment by stephenwaite** [comment]
whoops, squiz needs to be >3.3, beat on ole travis enough for today :sweat:

**Comment by bradymiller** [comment]
nice work! isn't devops fun :)

**Comment by stephenwaite** [comment]
PHP_CodeSniffer release 3.3.0 is where the errors begin, https://github.com/squizlabs/PHP_CodeSniffer/blob/7b0cc2c9d77fec932f2552024e592a43481a2af2/package.xml#L147

but we can add T_OPEN_TAG and T_CLOSE_TAG to get around inline html errors

**Comment by stephenwaite** [comment]
phpcs.xml file is only indented 2 spaces :)

**Comment by stephenwaite** [comment]
this PR is kind of a proof of concept showing how much progress squiz has made with PSR2 and how we can minimize the damage by adding excludes; then when feeling good incrementally remove the ignores and can run phpcbf to bring in the correctly formatted code

**Comment by stephenwaite** [comment]
on second thought might as well bite the bullet and accept what squiz recommends

File: ci/phpcs.xml [hunk 1]
```diff
@@ -19,16 +19,27 @@
...
   <exclude-pattern>*/library/classes/PDF_Label.php</exclude-pattern>
+  <exclude-pattern>*/node_modules/*</exclude-pattern>
+  <exclude-pattern>*/portal/assets/*</exclude-pattern>
+  <exclude-pattern>*/portal/patient/fwk/*</exclude-pattern>
+  <exclude-pattern>*/portal/patient/libs/*</exclude-pattern>
+  <exclude-pattern>*/portal/patient/templates/*</exclude-pattern>
+  <exclude-pattern>*/portal/patient/scripts/*</exclude-pattern>
+  <exclude-pattern>*/portal/patient/styles/*</exclude-pattern>
+  <exclude-pattern>*/public/assets/*</exclude-pattern>
+  <exclude-pattern>*/src/FHIR/*</exclude-pattern>
```
Comment (bradymiller) [inline ci/phpcs.xml]: how crazy do things get if you remove all the portal excludes and src/FHIR exclude?

**Comment by bradymiller** [comment]
It is really odd that is is fixing javascript files??

**Comment by stephenwaite** [comment]
didn't realize that was new since their subtitle states "PHP_CodeSniffer tokenizes PHP, JavaScript and CSS files and detects violations of a defined set of coding standards."

portal done; looks like it wants to fix formatting for FHIR in the docblocks

File: interface/main/calendar/modules/PostCalendar/pnincludes/overlib_mini.js [hunk 2]
```diff
@@ -241,514 +309,718 @@ var fnRef;
...
+    }
+    if (ns6) {
+        ns6=false;
+    }
+}
+if ((ns4)||(ie4)||(ns6)) {
+    document.onmousemove=mouseMove
+    if (ns4) {
+        document.captureEvents(Event.MOUSEMOVE)
+    } } else {
```
Comment (bradymiller) [inline interface/main/calendar/modules/PostCalendar/pnincludes/overlib_mini.js]: Thought I found an error here with 2 brackets, but I didn't. Like trying to beat a computer in chess; not possible :)

File: interface/main/calendar/modules/PostCalendar/pnincludes/overlib_mini.js [hunk 3]
```diff
@@ -241,514 +309,718 @@ var fnRef;
...
+            oframe=ol_frame;
+            if (ns4) {
+                over=oframe.document.overDiv
+                if (ie4) {
+                    over=oframe.overDiv.style
+                    if (ns6) {
+                        over=oframe.document.getElementById("overDiv");
+                    }
+                }
+            }
```
Comment (bradymiller) [inline interface/main/calendar/modules/PostCalendar/pnincludes/overlib_mini.js]: oh, wait, I think I found broken code. I am pretty sure this shouldn't be nested.

Comment (stephenwaite) [reply interface/main/calendar/modules/PostCalendar/pnincludes/overlib_mini.js]: okay, think i forgot to use the ci.sh script to select files locally and instead fed it whole directories; will explore tomorrow

Comment (stephenwaite) [reply interface/main/calendar/modules/PostCalendar/pnincludes/overlib_mini.js]: trying 
`$HOME/.composer/vendor/bin/phpcs -p -n --extensions=php,inc --report-diff=changes.diff --standard=ci/phpcs.xml .`
then can 
`patch -p0 -ui changes.diff`

Comment (stephenwaite) [reply interface/main/calendar/modules/PostCalendar/pnincludes/overlib_mini.js]: patch command throws an error will try instead
`$HOME/.composer/vendor/bin/phpcbf -p -n --extensions=php,inc --standard=ci/phpcs.xml .`

**Comment by stephenwaite** [comment]
will do a take 2

**Pull request closed without merging**